raiden/tests/fixtures/blockchain.py

```diff
@@ -50,9 +50,18 @@ def _token_addresses(
         token_amount,
         number_of_tokens,
         deploy_service,
-        blockchain_services,
+        participants,
         register):
-
+    """ Deploy `number_of_tokens` ERC20 token instances with `token_amount` minted and
+    distributed among `blockchain_services`. Optionally the instances will be `register`ed with
```

Comment:
I like highlighting arguments in the docstring this way. But I changed it to your preference.